webapp/makefile

```diff
@@ -23,17 +23,17 @@ build-docker: ./Dockerfile
 		-t $(DOCKER_REGISTRY)/$(IMAGE_NAME_WEBAPP):$(VERSION) --target run-stage \
 		-t $(DOCKER_REGISTRY)/$(IMAGE_NAME_WEBAPP):$(LATEST_TAG) --target run-stage \
 		--build-arg port="$(PORT)" \
-		--build-arg react_app_title="$(REACT_APP_TITLE)" \
-		--build-arg react_app_logo="$(REACT_APP_LOGO)" \
-		--build-arg react_app_version="$(REACT_APP_VERSION)" \
-	  --build-arg react_app_eos_api_host="$(REACT_APP_EOS_API_HOST)" \
-		--build-arg react_app_eos_api_port="$(REACT_APP_EOS_API_PORT)" \
-		--build-arg react_app_eos_api_protocol="$(REACT_APP_EOS_API_PROTOCOL)" \
-		--build-arg react_app_eos_chain_id="$(REACT_APP_EOS_CHAN_ID)" \
-		--build-arg react_app_hasura_url="$(REACT_APP_HASURA_URL)" \
-		--build-arg react_app_mapbox_access_token="$(REACT_APP_MAPBOX_ACCESS_TOKEN)" \
-		--build-arg react_captcha_key="$(REACT_APP_CAPTCHA-KEY)" \
-		--build-arg react_app_block_explorer_url="$(REACT_APP_BLOCK_EXPLORER_URL)" \
+		--build-arg REACT_APP_TITLE="$(REACT_APP_TITLE)" \
```

Comment:
we usually dont use upper case 
for example:
https://github.com/eoscostarica/gGoods/blob/3b0f8686883175bdbceebf8f7b398be50c5fe2f7/webapp/makefile#L26

Reply:
@xavier506 according to @kuronosec recommendation, we need to put those variables in upper_case or lower_case all of them because docker is sensitive case, so I decided to use upper_case because is more used in the whole project.

Reply:
@xavier506 What you recommend?

Reply:
@kuronosec things the different case sensitive could be the Lifebank error

Reply:
@leisterfrancisco I think its easier to read and more standard regarding the other projects, to leave args in lower case and envs in upper case.